ParisBNF/abb/BNFabb16.xml

```diff
@@ -0,0 +1,563 @@
+                            <title type="complete" ref="LIT3150Zechar"/>
+                            </msItem>
+
+                            <msItem xml:id="ms_i2.12">
+                            <locus from="39ra" to="40va"/>
+                            <title type="complete" ref="LIT3151Malachi"/>
+                            </msItem>
+                            </msItem>
+
+                            <msItem xml:id="ms_i3">
+                            <locus from="41ra" to="62ra"/>
+                            <title type="complete" ref="LIT1672Isaiah"/>
+                            </msItem>
+
+                            <msItem xml:id="ms_i4">
+                              <locus from="62ra" to="85vc"/>
+                              <title type="complete" ref="LIT5832BooksSalomon"/>
+                              <note>Books of Salomon.</note>
+
+                            <msItem xml:id="ms_i4.1">
+                            <locus from="62rb" to="72vc"/>
+                            <title type="complete" ref="LIT2195Prover"/>
+
+                            <msItem xml:id="ms_i4.1.1">
+                            <locus from="62rb" to="70vc"/>
+                            <title type="complete" ref="LIT3927Messale" xml:lang="gez"/>
+                            </msItem>
+
+                              <msItem xml:id="ms_i4.1.2">
+                              <locus from="70vс" to="72rc"/>
+                              <title type="complete" ref="LIT2396Tagsas" xml:lang="gez"/>
+                              <textLang mainLang="gez"/>
+                            </msItem>
+                            </msItem>
+
+                            <msItem xml:id="ms_i4.2">
+                            <locus from="72va" to="80vb"/>
+                            <title type="complete" ref="LIT2516Wisdom"/>
+                            <note>Wisdom of Solomon.</note>
+                            </msItem>
+
+                            <msItem xml:id="ms_i4.3">
+                              <locus from="80vb" to="84va"/>
+                              <title type="incomplete" ref="LIT1320Eccles"/>
+                                </msItem>
+                              </msItem>
+
+                              <msItem xml:id="ms_i5">
+                               <locus from="86ra" to="93vc"/>
+                               <title type="complete" ref="LIT1259Daniel"/>
+                              </msItem>
+
+                              <msItem xml:id="ms_i6">
+                               <locus from="94ra" to="112rc"/>
+                               <title type="complete" ref="LIT2358Sirach">
+                               <hi rend="rubric">ዘኢያሱ።</hi></title>
+                               <colophon xml:id="coloph1" xml:lang="gez">
+                                 <locus target="#112rc"/> >ተፈጸመ᎓ በዝየ᎓ ጥበቢሁ᎓
+                               ለኢያሱ᎓ ወልደ᎓ ሲራክ᎓ ዘተጽሕፈ᎓ በ<date>፷፻ወ፱፻ወ፸᎓ ዓመት</date>፨
```

Comment:
If I understand you well, this date most probably does not work out for the original; the copyist copied this date probably his "antigraph".

Reply:
To check whether I understand you well: The date given here, ፷፻ወ፱፻ወ፸᎓ ዓመት, probably refers to the date of production of this manuscript's antigraph? In this case, I still think that it could be marked up in the text and then explained as in the note that you gave, but of course you can decide this.